### ISSUE TITLE
`livenessEndpoint` and `readinessEndpoint` doesn't work

### ISSUE THREAD

**Issue by TheOrdinaryWow** [qlaffont/graceful-server-elysia#31]
as described in title.

reference:
 - https://github.com/qlaffont/graceful-server-elysia/blob/b2237316fec4d8c6009d170024dbe3f8532616ce/src/index.ts#L46-L48
 - https://github.com/qlaffont/graceful-server-elysia/blob/b2237316fec4d8c6009d170024dbe3f8532616ce/src/index.ts#L67-L69

**Comment by qlaffont** [comment]
/live works but /ready no I check it

**Comment by qlaffont** [comment]
Everything works with last version : 
```js
import { Elysia } from "elysia";
import { pluginGracefulServer, setServerIsReady } from 'graceful-server-elysia';

const app = new Elysia().use(pluginGracefulServer({serverIsReadyOnStart: false})).get("/", () => "Hello Elysia").listen(3000);

setServerIsReady();

console.log(
  `🦊 Elysia is running at ${app.server?.hostname}:${app.server?.port}`
);
```

```json
{
  "name": "app",
  "version": "1.0.50",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "dev": "bun run --watch src/index.ts"
  },
  "dependencies": {
    "elysia": "latest",
    "graceful-server-elysia": "^1.0.23"
  },
  "devDependencies": {
    "bun-types": "latest"
  },
  "module": "src/index.js"
}
```

Maybe for /ready, you didn't add serverIsReadyOnStart OR Add setServerIsReady

**Comment by qlaffont** [comment]
I have fixed it with 1.0.25 !
